Dockerfile

```diff
@@ -0,0 +1,96 @@
+
+# Adds DeepX Gazebo Rendering image.
+#
+# Arguments:
+# - BASE_IMAGE: Base image.
+# - TAG: Tag of the BASE_IMAGE.
+#
+# Docker commands:
+# - To build the image:
+#   docker build --build-arg BASE_IMAGE=ghcr.io/deepx-inc/base_images --build-arg TAG=foxy -t deepx-gz-rendering .
+# - To publish the image:
+#   docker push ghcr.io/deepx-inc/deepx-gz-rendering
+
+
+ARG BASE_IMAGE
+ARG TAG
+ARG FROM_SOURCE
+FROM ${BASE_IMAGE}:${TAG} AS DEEPX_GZ_RENDERING
+
+# ---BUILD TOOLS---
+RUN apt -qq update \
+    && apt -qq install -y --no-install-recommends \
+    cmake \
+    ffmpeg \
+    freeglut3-dev \
+    g++-8 \
+    git \
+    gnupg \
+    libfreeimage-dev \
+    libglew-dev \
+    libogre-2.1-dev \
```

Comment:
This needs to be `libogre-2.2-dev`. We also made this change in the gazebo dockerfile.

Reply:
Updated to `libogre-2.2-dev`.